tests/phpunit/integration/tests/Admin/Admin.php

```diff
@@ -118,6 +118,27 @@ public function test_prefill_post_content(): void {
 		$this->assertStringContainsString( $poster, $result );
 	}
 
+	/**
+	 * @covers ::prefill_post_content
+	 */
+	public function test_prefill_post_content_url_scheduled(): void {
```

Comment:
This test is not currently working because

1. The post is missing a `post_date` in the future
2. It uses `get_post( $story_id )` instead of `get_post( self::$post_id )`
3. The assertions are not precise enough

This should work:

```php
/**
 * @covers ::prefill_post_content
 */
public function test_prefill_post_content_url_scheduled(): void {
	wp_set_current_user( self::$admin_id );

	$story_id = self::factory()->post->create(
		[
			'post_type'    => \Google\Web_Stories\Story_Post_Type::POST_TYPE_SLUG,
			'post_status'  => 'future',
			'post_date'    => ( new DateTime( '+1day' ) )->format( 'Y-m-d H:i:s' ),
			'post_title'   => 'Example',
			'post_content' => '<html><head></head><body><amp-story standalone="" publisher="Web Stories" title="Example" poster-portrait-src="https://example.com/image.png"></amp-story></body></html>',
		]
	);

	$_GET['from-web-story'] = $story_id;
	$result                 = $this->instance->prefill_post_content( 'current', get_post( self::$post_id ) );

	$this->assertStringNotContainsString( '"url":"http://example.org/?post_type=web-story&#038;p=7', $result );
	$this->assertStringContainsString( '"url":"http://example.org/?post_type=web-story&p=7', $result );
}
```